src/Modules/Extensions/DependencyInjection/ExtensionsExtension.php

```diff
@@ -28,7 +28,8 @@ private function configureLiipImagine(ContainerBuilder $container): void
         foreach (Application::cases() as $application) {
             $application = ucfirst($application->value);
             foreach (InstalledModules::fromContainer($container)() as $moduleName) {
-                $dataRoots[$moduleName . 'Module' . $application] = $modulesDirectory . $moduleName . '/assets/' . $application . '/public';
+                $dataRoots[$moduleName . 'Module' . $application] = $modulesDirectory . $moduleName . '/assets/' .
```

Comment:
I would then rather move the key to a variable and keep the directory string together

Reply:
@bjorvack this has not been processed?

Reply:
fixed 6aea801fad4f313ef83101f0b71e948daad50765